mkdocs.yml

```diff
@@ -19,17 +19,8 @@ nav:
       - Getting started: getting-started/index.md
       - Tutorial:
         - Tutorial: getting-started/tutorial/index.md
-        - Creating an OpenSAFELY project: getting-started/tutorial/creating-an-opensafely-project/index.md
       - How-to guides:
         - How-to guides: getting-started/how-to/index.md
-        - Running OpenSAFELY in GitHub Codespaces: github-codespaces.md
-        - Installing the OpenSAFELY CLI: opensafely-cli.md
-        - Using GitHub and Git: install-github-and-git.md
-        - Installing Python: install-python.md
-        - Installing Docker: install-docker.md
-        - Installation on Windows: install-windows.md
-        - Installation on macOS: install-macos.md
-        - Installation on Linux: install-linux.md
```

Comment:
Having just looked for these pages in the table of contents on the branch I'm working on: was the intent to remove these pages entirely from the table of contents?

It feels a bit odd having them exist, but hidden.

Reply:
Yes, that was the intent. I don't think the table of contents needs to be a sitemap; more an overview of structure and useful/interesting content. Also, if every page has an entry in the table of contents, then the previous and next shortcut keys (<kbd>,</kbd> and <kbd>.</kbd>) force the reader into a depth-first search.